core/src/main/java/org/dozer/DozerBeanMapperBuilder.java

```diff
@@ -264,6 +270,36 @@ public DozerBeanMapperBuilder withSettingsProcessor(SettingsProcessor processor)
     }
 
     /**
+     * Registers a {@link ELEngine} for the mapper.
+     * Which can be used to resolve expressions within the defined mappings.
+     * <p>
+     * By default, {@link NoopELEngine} is registered.
```

Comment:
Would it be possible to create "smart" default here? Just like Spring fallows do, i.e. if GL implementation is available in classpath, create `DefaultELEngine`, otherwise fallback to `NoopELEngine`.

Reply:
Updated as requested. See: ExpressionFactory